crypto_box/src/lib.rs

```diff
@@ -181,6 +181,12 @@ use xsalsa20poly1305::aead::{
 use xsalsa20poly1305::XSalsa20Poly1305;
 use zeroize::{Zeroize, Zeroizing};
 
+#[cfg(feature = "seal")]
```

Comment:
This is a little more futureproof:

```suggestion
#[cfg(feature = "alloc")]
```

Reply:
I had it that way, but it failed CI with “unused extern crate” when `alloc` is enabled but not `seal`. Would you prefer an #[allow] for that lint on that line?

Reply:
Aah, I guess it's fine as-is then